packages/react-components/react-motion/library/src/hooks/useAnimateAtoms.ts

```diff
@@ -19,10 +22,10 @@ function useAnimateAtomsInSupportedEnvironment() {
           fill: 'forwards',
 
           ...params,
-          ...(isReducedMotion && { duration: 1 }),
+          ...((isReducedMotion || !SUPPORTS_PERSIST) && { duration: 1 }),
         });
 
-        animation.persist();
+        SUPPORTS_PERSIST && animation.persist();
```

Comment:
How about we just create a polyfill for https://developer.mozilla.org/en-US/docs/Web/API/Animation/commitStyles?

```ts
        if (SUPPORTS_PERSIST) {
          animation.persist();
        } else {
          const resultKeyframe = keyframes[keyframes.length - 1];
          Object.assign(element.style, resultKeyframe);
        }
```

This would ensure that we don't have weirdness when multiple animations are composed https://stackblitz.com/edit/typescript-hqxwu6?file=index.ts